XNAControls/XNATabControl.cs

```diff
@@ -52,6 +52,20 @@ public Color TextColor
             set { _textColor = value; }
         }
 
+        private Color? _textColorDisabled;
+
+        public Color TextColorDisabled
+        {
+            get
+            {
+                if (_textColorDisabled.HasValue)
+                    return _textColorDisabled.Value;
+
+                return UISettings.ActiveSettings.DisabledItemColor;
+            }
```

Comment:
Use the null-coalescing operator `??` to simplify the code: `return _textColorDisabled ?? UISettings.ActiveSettings.DisabledItemColor;`

While at it, you can make use of expression bodies to change the currently-7-line getter into the following nice and simple piece of code:
`get => _textColorDisabled ?? UISettings.ActiveSettings.DisabledItemColor;`